control/narrative_table.go

```diff
@@ -0,0 +1,91 @@
+package control
+
+import (
+	"errors"
+	"regexp"
+
+	"github.com/jbowtie/gokogiri/xml"
+	"github.com/opencontrol/fedramp-templater/docx/helper"
+	"github.com/opencontrol/fedramp-templater/opencontrols"
+)
+
+func findSectionKey(row xml.Node) (section string, err error) {
+	re := regexp.MustCompile(`Part ([a-z])`)
```

Comment:
Some narratives use `Req` instead of `Part` and numbers.

![image](https://cloud.githubusercontent.com/assets/7788930/18033500/e2f2f0d6-6cf4-11e6-8ac1-f58ec42dd853.png)